package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-plugin-unicorn": "^4.0.3",
     "istanbul": "^0.4.3",
     "jsdoc": "^3.6.3",
+    "jsonlint": "^1.6.3",
```

Comment:
Running `node Makefile.js` failed both on my machine and Travis without this dependency